### PR TITLE
Add certificate fetcher and displayer

### PR DESCRIPTION
**Pull request by junron**
[Task](https://codein.withgoogle.com/dashboard/task-instances/5891517274128384/)

Fetches student certs and lists them in a table in descending order by date issued.

### Thumbnail mode:
cert-fetcher-thumbnail

### List mode:
cert-fetcher-list

Did not make any changes to other components except the student component

### Improvements

- [X]  Use AG-grid
- [X] Removed unnecessary classes
- [X] Thumbnail mode

### CODE REVIEW

**Comment by junron** [comment]
Perhaps there could be a switch between thumbnail mode and list view

**Comment by nikitadhiman** [comment]
Yes @junron we are completely fine with list implementation because in case a student has like 100+ certificates he/she would definitely like list view. Please implement ag-grid for this, very powerful and feature rich

**Comment by junron** [comment]
Hmm I'll get to work asap

**Comment by junron** [comment]
Implemented AG grid for the list view mode. What would be a suitable icon for the certificate thumbnails? Should I use the Jboss logo or allow organizations to specify a logo url?

File: src/app/student/student.component.ts [hunk 1]
```diff
@@ -1,15 +1,46 @@
 import { Component, OnInit } from '@angular/core';
+import { StudentService } from "../services/student.service"
+import { Observable } from 'rxjs';
 
+class Cert{
+  issuer:string;
```
Comment (nikitadhiman) [inline src/app/student/student.component.ts]: no need of "class" here

File: src/app/student/student.component.ts [hunk 2]
```diff
@@ -1,15 +1,46 @@
...
+  date:string;
+  name:string;
+}
+
+class ColumnDefs {
+  headerName:string;
```
Comment (nikitadhiman) [inline src/app/student/student.component.ts]: same here

File: src/app/student/student.component.ts (hunk 1 above)
Comment (junron) [reply src/app/student/student.component.ts]: Ok I'll get rid of that.

Comment (junron) [reply src/app/student/student.component.ts]: Removed classes

File: src/app/student/student.component.ts (hunk 2 above)
Comment (junron) [reply src/app/student/student.component.ts]: Removed classes

**Comment by junron** [comment]
Added thumbnail mode and mode switch button. Could @nikitadhiman take another look?

**Comment by junron** [comment]
Can I base my submission for https://codein.withgoogle.com/dashboard/task-instances/5628665409830912/ on this PR? Could I use the same PR

**Comment by nikitadhiman** [comment]
closing this for now

**Pull request closed without merging**